app/webpacker/components/Panel/pages/RunValidatorsPage/CompetitionRangeSelector.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import { Form } from 'semantic-ui-react';
+import { QueryClient, useQuery } from '@tanstack/react-query';
+import UtcDatePicker from '../../../wca/UtcDatePicker';
+import getCompetitionList from './api/getCompetitionList';
+import Loading from '../../../Requests/Loading';
+import Errored from '../../../Requests/Errored';
+
+const RUN_VALIDATORS_QUERY_CLIENT = new QueryClient();
+const MAX_COMPETITIONS_PER_QUERY = 50;
+
+export default function CompetitionRangeSelector({ range, setRange }) {
+  const [startDate, setStartDate] = useState(range?.startDate);
+  const [endDate, setEndDate] = useState(range?.endDate);
+
+  const enableCompetitionListFetch = Boolean(startDate && endDate);
```

Comment:
I'm strongly in favour of naming things based on what they are, not how they are used.
```suggestion
  const bothDatesAreSelected = Boolean(startDate && endDate);
```

Reply:
Thanks for the suggestion, changed it.